ssl/ssl_locl.h

```diff
@@ -993,6 +995,10 @@ struct ssl_st {
     EVP_CIPHER_CTX *enc_write_ctx; /* cryptographic state */
     unsigned char write_iv[EVP_MAX_IV_LENGTH]; /* TLSv1.3 static write IV */
     EVP_MD_CTX *write_hash;     /* used for mac generation */
+
```

Comment:
I prefer them - I think it helps readability. But I don't care enough about it, so I removed them.